tests/Feature/Api/UserMembershipTest.php

```diff
@@ -0,0 +1,49 @@
+<?php
+
+namespace Tests\Feature\Api;
+
+use Nuwave\Lighthouse\Testing\MakesGraphQLRequests;
+use Tests\TestCase;
+
+class UserMembershipTest extends TestCase
+{
+    use MakesGraphQLRequests;
+
+    public function testCanGiveVisitingMembershipToUser()
```

Comment:
Worth expecting an exception too?

Reply:
It doesn't throw an exception because GraphQL handles it. But I have made the exception a Public one so now it will return a validation error

Reply:
I think even if is caught you can still expect it’s thrown under the hood

Reply:
Probably, but this is probably a good thing? Shows there there is something wrong client side? If we handle it, it won't send the error back to the client so...

Reply:
Oh wait, no the exception isn't thrown under the hood, otherwise the test would fail